src/main.cc

```diff
@@ -178,6 +178,39 @@ signal_cb(ev_loop *loop, struct ev_signal *w, int revents)
 	tarantool_exit(0);
 }
 
+static void
+signal_sigint_cb(ev_loop *loop, struct ev_signal *w, int revents)
```

Comment:
Missing [change log entry](https://github.com/tarantool/tarantool/tree/master/changelogs/unreleased). Missing doc bot request. (We desperately need to fix #6790.)

Reply:
Fixed.